superset-frontend/stylesheets/superset.less

```diff
@@ -603,7 +603,3 @@ hr {
     top: 269px !important;
   }
 }
-
-.ant-dropdown.ant-dropdown-placement-bottomRight {
-  top: 133px !important;
```

Comment:
@pkdotson this was introduced by PR #14184. Not sure what side effects might have removing it, but in general it seems that the approach of forcing the positions might not be the right one in the first place.

The potential issue was also reported by @rusackas on this review comment https://github.com/apache/superset/pull/14184/files#r626190039 but was then overlooked. I would please ask you to have a look at that portion of the changes and find different solutions. I am pretty sure this is not the only issue we might face.

Reply:
thanks for the investigation!@geido @pkdotson how much refactoring left in this project? 

@jinghua-qa